README.md

```diff
@@ -12,125 +12,139 @@ Usage of this API is limited to registered users only. If you would like to use
 Run the following command to install the package using composer:
 
 ```
-$ composer require oberonamsterdam/tor-graphql-client
+$ composer require oberonamsterdam/tor-api-client
 ```
 
 # Usage
 
-To use this Client you need to define the API Key and endpoint as an .env variable or supply it when initiating the client class.
-```
-TOR_GRAPHQL_ENDPOINT="https://travelbase.nl/api/management/v2/graphql"
-TOR_GRAPHQL_APIKEY="MY_SECRET_APIKEY"
-```
+To use this Client you need to provide the API Key and endpoint when initiating the client class.
 
 ```php
-$client = new \TOR\GraphQL\TorClient("https://example.com", "MY_SECRET_APIKEY");
+$client = new \Oberon\TorClient\ApiClient("https://travelbase.nl", "APIKEY");
 ```
 
 # Example calls
 Retrieve a collection of all partners:
 ```php
-$client = new \TOR\GraphQL\TorClient();
-/** @var \TOR\GraphQL\Model\Partner[] $partners */
+$client = new \Oberon\TorClient\ApiClient("https://travelbase.nl", "APIKEY");
+/** @var \Oberon\TorClient\Model\Partner[] $partners */
 $partners = $client->getPartners();
 ```
   
 Retrieve a single partner:
 ```php
-$client = new \TOR\GraphQL\TorClient();
-/** @var \TOR\GraphQL\Model\Partner $partner */
+$client = new \Oberon\TorClient\ApiClient("https://travelbase.nl", "APIKEY");
+/** @var \Oberon\TorClient\Model\Partner $partner */
 $partners = $client->getPartner($partnerId);
 ```
 
 Retrieve a single accommodation:
 ```php
-$client = new \TOR\GraphQL\TorClient();
-/** @var \TOR\GraphQL\Model\Accommodation $accommodation */
+$client = new \Oberon\TorClient\ApiClient("https://travelbase.nl", "APIKEY");
+/** @var \Oberon\TorClient\Model\Accommodation $accommodation */
 $accommodation = $client->getAccommodation($accommodationId);
 ```
 
 Retrieve a single rentalUnit:
 ```php
-$client = new \TOR\GraphQL\TorClient();
-/** @var \TOR\GraphQL\Model\RentalUnit $rentalUnit */
+$client = new \Oberon\TorClient\ApiClient();
```

Comment:
In de andere voorbeelden heb je de parameters toegevoegd, missen hier nog.

Reply:
- [x] toevoegen parameters in readme